### PR TITLE
Added console command: spawn to inventory

### PR DESCRIPTION
**Pull request by xrSimplDimp**
Let's use the console command to add a section to the inventory

### CODE REVIEW

File: src/xrGame/console_commands.cpp [hunk 1]
```diff
@@ -181,6 +181,33 @@ class CCC_MemStats : public IConsole_Command
 };
 
 // console commands
+class CCC_Spawn_to_inv : public IConsole_Command
+{
+public:
+    CCC_Spawn_to_inv(LPCSTR N) : IConsole_Command(N){};
+    virtual void Execute(LPCSTR args)
```
Comment (375gnu) [inline src/xrGame/console_commands.cpp]: better: `void Execute(LPCSTR args) override`

File: src/xrGame/console_commands.cpp [hunk 2]
```diff
@@ -181,6 +181,33 @@ class CCC_MemStats : public IConsole_Command
...
+    {
+        if (!g_pGameLevel)
+        {
+            Log("Error: No game level!");
+            return;
+        }
+
+        if (!pSettings->section_exist(args))
+        {
+            Msg("! Section [%s] isn`t exist...", args);
```
Comment (375gnu) [inline src/xrGame/console_commands.cpp]: doesn't exist

File: src/xrGame/console_commands.cpp [hunk 3]
```diff
@@ -181,6 +181,33 @@ class CCC_MemStats : public IConsole_Command
...
+            return;
+        }
+
+        char Name[128];
+        Name[0] = 0;
+        sscanf(args, "%s", Name);
```
Comment (375gnu) [inline src/xrGame/console_commands.cpp]: may i ask why do you need a local copy? and scanf is unsafe

File: src/xrGame/console_commands.cpp [hunk 4]
```diff
@@ -181,6 +181,33 @@ class CCC_MemStats : public IConsole_Command
...
+
+        Level().spawn_item(Name, Actor()->Position(), false, Actor()->ID());
+    }
+    virtual void Info(TInfo& I) { strcpy(I, "name,team,squad,group"); }
```
Comment (375gnu) [inline src/xrGame/console_commands.cpp]: `void Info(TInfo& I) override ...`
xr_strcpy

File: src/xrGame/console_commands.cpp (hunk 1 above)
Comment (xrSimplDimp) [reply src/xrGame/console_commands.cpp]: Well, I'll take this into account and change it to new methods

File: src/xrGame/console_commands.cpp (hunk 2 above)
Comment (xrSimplDimp) [reply src/xrGame/console_commands.cpp:198]: > не существует
I'll redo it

Comment (xrSimplDimp) [reply src/xrGame/console_commands.cpp]: > не существует

I'll redo it

**Pull request merged**